qubes/tests/integ/backupcompatibility.py

```diff
@@ -108,7 +108,6 @@
 Exec=qvm-run -q --tray -a %VMNAME% '{command} %u'
 Categories=Network;WebBrowser;
 X-Qubes-VmName=%VMNAME%
-Icon=%VMDIR%/icon.png
```

Comment:
This should stay, because this is how .desktop files looked at R2.0 times.
But I think that isn't a problem for removing the icon. Perhaps the restore code should adjust `Icon=` entry appropriately (but that's out of scope for this PR).